src/components/message/index.ts

```diff
@@ -0,0 +1,23 @@
+import tpl from './tpl.hbs';
+import Block from '../../service/Block';
+
+interface ITempMassage {
+    chatTitle: string;
+    href: string;
+    myMassage?: string;
+    chatMassage: string;
```

Comment:
Опечатка скорее всего
```suggestion
    myMessage?: string;
    chatMessage: string;
```